indexer/search_string_utils.cpp

```diff
@@ -402,6 +402,36 @@ class StreetsSynonymsHolder
   Trie m_strings;
 };
 
+class StreetsDirectionsHolder
+{
+  std::vector<UniString> m_strings;
+
+public:
+  StreetsDirectionsHolder()
+  {
+    // Do *not* change the order.
+    for (auto const * s : {"n", "north", "s", "south", "w", "west", "e", "east",
+                           "ne", "northeast", "nw", "northwest", "se", "southeast", "sw", "southwest" })
+    {
+      m_strings.emplace_back(NormalizeAndSimplifyString(s));
```

Comment:
1. Is normalization and simplification really needed here?
2. Does it make sense to add a UniString(char32_t const * u32str) constructor and initialize like this:

```
m_strings {U"n", U"north", ...}
```